src/Application/Authorizers/LaravelImportBusinessAuthorizer.php

```diff
@@ -71,12 +71,17 @@ public function authorize(array $data, Business $business = null)
             throw new ImportBusinessUnauthorizedException('User is a guest.');
         }
 
-        if ($this->isUserRestarter($user) && $this->dataIsPublished($data)) {
-            throw new ImportBusinessUnauthorizedException('User is a restarter and cannot publish businesses');
-        }
+        // Normal restarters can't edit.
+        $editallowed = $user->isEditor() || !$user->isRegionalAdmin() || !$user->isSuperAdmin();
```

Comment:
Is this right?  Seems to be saying that edit is allowed if user is not a regional admin, or not a super admin.

Reply:
No, it's wrong.  Fix applied.